analyzer_test.go

```diff
@@ -0,0 +1,14 @@
+package wraperr_test
+
+import (
+	"testing"
+
+	"golang.org/x/tools/go/analysis/analysistest"
+
+	"github.com/srvc/wraperr"
+)
+
+func TestAnalyzer(t *testing.T) {
+	testdata := analysistest.TestData()
+	analysistest.Run(t, testdata, wraperr.Analyzer, "simple")
```

Comment:
**[unparam]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
cannot use wraperr.Analyzer (variable of type *golang.org/x/tools/go/analysis.Analyzer) as *invalid type value in argument to analysistest.Run